src/App.jsx

```diff
@@ -1,9 +1,33 @@
-import { useState } from 'react'
+import { useState, useEffect} from 'react'
+import axios from "axios"
 import './App.css'
+import Header from './Components/Header'
+import NavWrapper from './Components/NavWrapper'
+import ContentWrapper from './Components/ContentWrapper'
+import BottomButtons from './Components/BottomButtons'
 
 function App() {
+
+  const[articles, setArticles] = useState([])
```

Comment:
The state should live in the lowest component possible, at the moment you are only passing articles down and it seems that the state could be there instead 🙂